payments-core/src/test/java/com/stripe/android/ApiKeyFixtures.kt

```diff
@@ -15,6 +15,7 @@ internal object ApiKeyFixtures {
     const val BANCONTACT_PUBLISHABLE_KEY = "pk_test_vOo1umqsYxSrP5UXfOeL3ecm"
     const val GIROPAY_PUBLISHABLE_KEY = "pk_test_vOo1umqsYxSrP5UXfOeL3ecm"
     const val EPS_PUBLISHABLE_KEY = "pk_test_vOo1umqsYxSrP5UXfOeL3ecm"
+    const val MULTIBANCO_PUBLISHABLE_KEY = "pk_test_51JtgfQKG6vc7r7YCU0qQNOkDaaHrEgeHgGKrJMNfuWwaKgXMLzPUA1f8ZlCNPonIROLOnzpUnJK1C1xFH3M3Mz8X00Q6O4GfUt"
```

Comment:
I think this is causing issues with my E2E test, probably need to pair with someone to debug